src/redux_module/selectedSlice.js

```diff
@@ -0,0 +1,26 @@
+/* eslint-disable no-param-reassign */
+
+import { createSlice } from '@reduxjs/toolkit';
+
+const selectedSlice = createSlice({
```

Comment:
```suggestion
const { actions, reducer } = createSlice({
```

destructuring 문법을 이용하시면 이렇게 작성하실 수도 있어요.

그러면 `export const { ... } = actions;` 처럼 작성이 가능하기 때문에 중간에 변수 이름을 고민할 필요가 없죠 ;)

Reply:
오 좋네요. 수정했습니다!